lib/Runtime/Library/JavascriptGeneratorFunction.cpp

```diff
@@ -86,6 +129,50 @@ namespace Js
         return generator;
     }
 
+    Var JavascriptGeneratorFunction::EntryAsyncFunctionImplementation(RecyclableObject* function, CallInfo callInfo, ...)
+    {
+        PROBE_STACK(function->GetScriptContext(), Js::Constants::MinStackDefault);
+        ARGUMENTS(stackArgs, callInfo);
+
+        ScriptContext* scriptContext = function->GetScriptContext();
+        JavascriptLibrary* library = scriptContext->GetLibrary();
+        DynamicObject* generatorPrototype = scriptContext->GetLibrary()->CreateGeneratorConstructorPrototypeObject();
```

Comment:
I think this object is not accessible since it is meant for the .prototype property of the async/generator virtual script function object.  We might be able to get away with putting javascript null here for both async and generator implementations.  Otherwise we're getting wasted allocations for every invocation of generator and async functions.

I think, please confirm.